specreduce/extract.py

```diff
@@ -102,6 +102,12 @@ def _ap_weight_image(trace, width, disp_axis, crossdisp_axis, image_shape):
     return wimage
 
 
+def _to_spectrum1d_pixels(fluxes):
```

Comment:
I agree that we'll likely want/need to do that once we have wavelength calibration.  But since this is an internal function, we can extend that at any point in the future.